strawberryfields/apps/qchem/utils.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+This module contains various utility functions needed to perform quantum chemistry calculations with
+Strawberry Fields.
+"""
+from typing import Tuple
+
+import numpy as np
+from scipy.constants import c, h, m_u, pi
+
+
+def duschinsky(
+    Li: np.ndarray, Lf: np.ndarray, ri: np.ndarray, rf: np.ndarray, wf: np.ndarray, m: np.ndarray
+) -> Tuple[np.ndarray, np.ndarray]:
+    r"""Generate the Duschinsky rotation matrix and displacement vector.
```

Comment:
Maybe worth adding a bit more information here about how they are computed and what the inputs are

Reply:
Done.